### PR TITLE
Fix Publish Queue expiry

### PR DESCRIPTION
**Pull request by naspeh**
[SDESK-1816](https://dev.sourcefabric.org/browse/SDESK-1816)

### CODE REVIEW

File: apps/archive/commands.py [hunk 1]
```diff
@@ -28,6 +28,16 @@
 logger = logging.getLogger(__name__)
 
 
+def log_exeption(fn):
```
Comment (petrjasek) [inline apps/archive/commands.py]: I don't get why it's needed, there is no logging otherwise?

Comment (petrjasek) [reply apps/archive/commands.py]: should be logged here https://github.com/superdesk/superdesk-core/blob/master/superdesk/__init__.py#L57-L64

Comment (naspeh) [reply apps/archive/commands.py]: we have two calls
```
+        self._remove_expired_publish_queue_items()
 +        self._remove_expired_items(now)
```
in previous version if the first fails second one never call, so I wrapped these function to be sure that second function will be called

Comment (petrjasek) [reply apps/archive/commands.py]: I see, would be good imo to put this into the comment

Comment (naspeh) [reply apps/archive/commands.py]: comment is here
```diff
+        # next functions wrapped with log_exeption, so no try there
```

File: apps/archive/commands.py [hunk 2]
```diff
@@ -40,18 +51,20 @@ def run(self):
...
-        try:
-            logger.info('{} Removing expired content for expiry.'.format(self.log_msg))
-            self._remove_expired_items(now)
-            self._remove_expired_publish_queue_items()
-        finally:
-            unlock(lock_name)
+
+        logger.info('{} Removing expired content for expiry.'.format(self.log_msg))
+        # both functions should be called, even the first one throw exception,
+        # so they are wrapped with log_exeption
```
Comment (naspeh) [inline apps/archive/commands.py]: here is I updated comment with better description

**Pull request merged**